getting-started/Installation/configuration.md

```diff
@@ -55,7 +55,7 @@ end
 
 ### Where do custom settings go?
 
-You can include new settings withing your environment yaml file by specifying them in the secrets section of the yaml.
+You can include new settings within your environment YAML file by specifying them in the secrets section of the YAML.
```

Comment:
You can include new settings in any of the environment YAML files by specifying them in the secrets section.